xline/src/storage/db.rs

```diff
@@ -117,13 +90,10 @@ where
                 let key = new_rev.encode_to_vec();
                 let value = del_kv.encode_to_vec();
                 let _prev_val = storage.insert(key, value).map_err(|e| {
-                    ExecuteError::InvalidCommand(format!(
-                        "Failed to insert revision {:?}: {}",
-                        new_rev, e
-                    ))
-                });
-                Ok(del_kv)
-            })
-            .collect()
+                    ExecuteError::db_error(format!("Failed to insert key-value, error: {e}"))
```

Comment:
It looks that there is nothing different between `ExecuteError::db_error(format!("Failed to insert key-value, error: {e}"))` and `ExecuteError::DbError(format!("Failed to insert key-value, error: {e}"))`. So why do need such a function?

Reply:
removed